components/ai_chat/core/browser/tools/tool_provider.cc

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/ai_chat/core/browser/tools/tool_provider.h"
+
+namespace ai_chat {
+
+ToolProvider::ToolProvider() = default;
+
+ToolProvider::~ToolProvider() = default;
```

Comment:
nit: I guess we're adding more to this file in the future so this .cc exists and not just in header?